app/routes/payWithInterledger.tsx

```diff
@@ -0,0 +1,215 @@
+import { conform, useForm } from '@conform-to/react'
+import { getFieldsetConstraint, parse } from '@conform-to/zod'
+import { type WalletAddress } from '@interledger/open-payments'
+import {
+  type ActionFunctionArgs,
+  json,
+  redirect,
+  type LoaderFunctionArgs
+} from '@remix-run/node'
+import { Form, useActionData, useLoaderData } from '@remix-run/react'
+import { z } from 'zod'
+import { Header } from '~/components/header'
+import Quote from '~/components/quoteDialog'
+import { Button } from '~/components/ui/button'
+import { Field } from '~/components/ui/form/form'
+import { PayWithInterledgerMark } from '~/components/ui/logo'
+import { useDialogContext } from '~/lib/context/dialog'
+import { fetchQuote, initializePayment } from '~/lib/open-payments.server'
+import { getValidWalletAddress } from '~/lib/validators.server'
+import { commitSession, destroySession, getSession } from '~/session'
+import { formatAmount } from '~/utils/helpers'
+
+export async function loader({ request }: LoaderFunctionArgs) {
+  const searchParams = new URL(request.url).searchParams
+  const isQuote = searchParams.get('quote') || false
+  const receiver = searchParams.get('receiver') || ''
+
+  const session = await getSession(request.headers.get('Cookie'))
+
+  let receiverName = ''
+  let receiveAmount = null
+  let debitAmount = null
+
+  if (isQuote) {
```

Comment:
Here we should check if the receiver or the quote is set. Navigating to the page just shows a payment page without the ability to add a payment pointer? Redirect to the payment page maybe?